rust/protocol/src/identity_key.rs

```diff
@@ -67,6 +67,11 @@ impl IdentityKeyPair {
         }
     }
 
+    pub fn from_private_key(private_key: curve::PrivateKey) -> Result<Self> {
```

Comment:
Should this be a TryFrom implementation?

Reply:
It could just be `From` since it can't fail

Reply:
There is some other cleanups that should happen in the curve code - I will defer this to a future PR (today) since it doesn't really have anything to do with the TS API